plugins/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "appium-inspector-plugin",
+  "version": "2024.12.1",
+  "description": "An app inspector for use with an Appium server",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/appium/appium-inspector.git"
+  },
+  "author": {
+    "name": "Appium Developers",
+    "url": "https://github.com/appium"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/appium/appium-inspector/issues"
+  },
+  "keywords": [
+    "appium"
+  ],
+  "homepage": "https://github.com/appium/appium-inspector",
+  "main": "index.js",
+  "type": "module",
+  "peerDependencies": {
+    "appium": "^2.0.0"
+  },
+  "files": [
+    "index.js",
+    "package.json",
+    "dist-browser",
+    "README.md"
+  ],
+  "dependencies": {},
+  "devDependencies": {
+    "@appium/base-plugin": "^2.2.50"
```

Comment:
Is this a dev dependency?

Reply:
@mykola-mokhnach Good catch! Actually we don't need this in dependencies nor devDependecies as it comes from peerDependencies 

We can use as `import {BasePlugin} from 'appium/plugin';`

Reply:
@mykola-mokhnach When changed to the above command, we see 

[Appium] Could not load plugin 'inspector', so it will not be available. Error in loading the plugin was: Cannot find module '/Users/saikrishna/Documents/git/appium-inspector/node_modules/appium/plugin' imported from /Users/saikrishna/Documents/git/appium-inspector/plugins/index.js
Did you mean to import "appium/plugin.js"?
[Appium] Error: Cannot find module '/Users/saikrishna/Documents/git/appium-inspector/node_modules/appium/plugin' imported from /Users/saikrishna/Documents/git/appium-inspector/plugins/index.js
Did you mean to import "appium/plugin.js"?